### PR TITLE
Mandos section redesign & update

### PR DESCRIPTION
**Pull request by andrei-marinica**
#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category(categories) does this pull request belong to?
- [x] document new feature
- [x] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/developers/mandos-reference/values-simple.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+`"address:my_address"` is the same as:
+- `"str:my_address______________________"` or
+- `"0x6d795f616464726573735f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f"`.
+:::
+
+## **Smart Contract Addresses**
+
+`sc:` constructs a dummy smart contract address. 
+
+On Elrond, smart contract addresses have a different format than user addresse - they start with 8 bytes of zero.
```
Comment (andrei-diaconescu) [inline docs/developers/mandos-reference/values-simple.md:165]: typo

**Review by andrei-diaconescu — CHANGES_REQUESTED** [review]

File: docs/developers/mandos-reference/values-complex.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+This is an abridged section of the actual lottery contract in the examples.
+
+```json
+{
+    "step": "checkState",
+    "accounts": {
+        "sc:lottery": {
+            "storage": {
+                "str:lotteryInfo|nested:str:lottery_name": {
```
[the remark this answers is not in the document]
Comment (dorin-iancu) [reply docs/developers/mandos-reference/values-complex.md:93]: It is :)

File: docs/developers/mandos-reference/values-complex.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+:::
+
+:::tip
+
+Once again, note that all contained values are in [nested encoding format](/developers/developer-reference/elrond-serialization-format#the-concept-of-top-level-vs-nested-objects):
+- the token identifier has its length automatically prepended by the `nested:` prefix,
+- big ints are given with the `biguint:` syntax, which prepends their byte length,
+- small ints are given in full length,
+- lists have their length explicitly encoded at the start.
```
Comment (dorin-iancu) [inline docs/developers/mandos-reference/values-complex.md:135]: Maybe a small note here about how all lengths are encoded on 4 bytes

**Comment by dorin-iancu** [comment]
Great work :tada:

**Review by dorin-iancu — APPROVED** [review]

**Review by andrei-diaconescu — APPROVED** [review]

**Pull request merged**